test/namespace/lib.v3

```diff
@@ -0,0 +1,9 @@
+class Util {
```

Comment:
I think we can test this by just having another application in `apps/`

Reply:
I added it to `bench/`